test-data/unit/plugins/named_callable.py

```diff
@@ -0,0 +1,28 @@
+from mypy.plugin import Plugin
+from mypy.types import CallableType
+
+
+class MyPlugin(Plugin):
+    def get_function_hook(self, fullname):
+        if fullname == 'm.decorator1':
+            return decorator_call_hook
+        if fullname == 'm._decorated':
```

Comment:
Aarg. It took me on a wild goose chase figuring out where this name was defined, until I finally realized it's done in `decorator_call_hook()` below. :-)